src/functionsfolder/commandSpells/lsSpell.js

```diff
@@ -1,7 +1,8 @@
 import {mazeArray} from '../../arrays/mazeArray';
 
-export default function lSpell(preparedInputAsArray, currentArrayPosition) {
+export default function lsSpell(preparedInputAsArray, currentArrayPosition, changeEnlighten) {
   if (preparedInputAsArray.length === 1) {
+    changeEnlighten(true);
```

Comment:
Is `changeEnlighten` a function? If so, I would probably call it `onChangeEnlighten` when passing it as a callback function.